src/app/items/components/item/item.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Item } from 'src/entities/item.model';
+
+@Component({
+  selector: 'app-item',
+  templateUrl: './item.component.html',
+  styleUrls: ['./item.component.scss']
+})
+export class ItemComponent implements OnInit {
+  @Input() item: Item;
+  @Output() onDeleteItem = new EventEmitter<string>();
+  @Output() onEditItem = new EventEmitter<string>();
+  @Output() onViewItem = new EventEmitter<string>();
+  
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  deleteItem(): void {
+    this.onDeleteItem.emit(this.item.id);
+  }
+
+  viewItem(): void {
+    this.onViewItem.emit(this.item.id);
+  }
+
```

Comment:
consider emitting the item itself and then you will not need to find by id later